cmd/sidecar/main.go

```diff
@@ -20,6 +22,10 @@ var infoLogger = log.InfoLogger
 
 func main() {
 	var err error
+	var secretsHandler *secrets.Secrets
+
+	//SECRETS_DESTINATION read from env variables
```

Comment:
Add "TODO" comment to be clear how it should be implemented later

Reply:
removing because it doesn't contribute to anything